doc/depthFirstSearch/pgr_depthFirstSearch.rst

```diff
@@ -37,22 +36,20 @@ Undirected DFS algorithm implemented by Boost.Graph.
 Description
 -------------------------------------------------------------------------------
 
-Depth First Search algorithm is a well known traversal algorithm which starts
-from a root vertex (``Root vid``) and visits all the nodes in a graph in the
-depth-first search traversal order. An optional non-negative maximum depth
-parameter (``max_depth``) can be specified to get the results up to a particular
-depth.
+Depth First Search algorithm is a traversal algorithm which starts from a root vertex
+and visits all the vertices in a graph, in the depth-first search traversal order.
```

Comment:
**in the depth-first search traversal order.**

I mentioned that you need to change this, because you are using what you are describing as part of the description.

Description of a "House"
A house is a house